app/server/utils/user.py

```diff
@@ -514,18 +518,23 @@ def proccess_create_or_modify_user_request(attribute_dict,
         use_precreated_pin=use_precreated_pin,
         use_last_4_digits_of_id_as_initial_pin=use_last_4_digits_of_id_as_initial_pin,
         existing_transfer_account=existing_transfer_account,
-        is_beneficiary=is_beneficiary, is_vendor=is_vendor, is_self_sign_up=is_self_sign_up
+        is_beneficiary=is_beneficiary, is_vendor=is_vendor, is_self_sign_up=is_self_sign_up,
+        token=token
     )
 
-    custom_attributes = set_custom_attributes(
-        attribute_dict, user)
+    if attribute_dict.get('custom_attributes', None) is None:
```

Comment:
uh, in general this file's changes should not be made...

Reply:
I am not on my machine atm but I can clean this up in 30 minutes or so.